src/OpenTracing/StartSpanOptions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+
+namespace OpenTracing
+{
+    /// <summary>
+    /// Provides a way to configure the behavior of <see cref="ITracer.StartSpan"/>.
+    /// </summary>
+    public class StartSpanOptions
```

Comment:
I'm not understanding why we want the core OpenTracing API to support both the builder and the StartSpanOptions... As with any API, it's easier to add things than to subtract them. If these two approaches are semantically equivalent, let's choose the one that's easier for the caller (the builder IMO) and let implementations create a StartSpanOptions or similar if that makes their lives easier internally.

Forgive me if there's something that the Options or Builder approach can do that the other cannot...